setup.py

```diff
@@ -22,8 +22,7 @@
         "toposort"
         ],
     include_package_data=True,
-    #scripts=['bin/qspector/qspector.py'] ,
     entry_points={
-        'console_scripts': ['QCCalculator = QCCalculator.cli:start']
+        'console_scripts': ['qccalculator = qccalculator.cli:start']
```

Comment:
This also changes the executables name to lower case. I'd prefer the exec to be called QCCalculator to distinguish from the python package. Same as in `ThermoRawFileParser.exe`:
`'console_scripts': ['QCCalculator = qccalculator.cli:start']`

Reply:
Done